deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -752,6 +752,10 @@ spec:
               description: Name of Scheduler to use for pod placement (defaults to
                 “default-scheduler”)
               type: string
+            serviceAccount:
```

Comment:
Does the `serviceAccount` default to anything if not set?  Should we document that here?

Reply:
If no custom serviceaccount is attached to a pod, a default service account assigned to the cluster is used by the pod. Let me document that.